modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/login-button/index.css

```
@@ -0,0 +1,21 @@
+.evp-login-button :hover {
```

Please, remove this space.
```suggestion
.evp-login-button:hover {
```

---

Sure. Adjusted!